pkg/telemetry/prometheus/node.go

```diff
@@ -97,6 +101,37 @@ func getMemoryStats() (memoryLoad float32, err error) {
 	return
 }
 
+func AddServiceOperation(typeStr string, status string, errType string) {
+	switch typeStr {
+	case "ice_connection", "peer_connection", "offer", "answer":
+		if errType == "" {
+			connectionSuccess.Inc()
```

Comment:
what is the goal we are trying to measure here? if it's a participant connection, then recording these would be drastically over-counting things.